### PR TITLE
Updated README.md to clarify complication install

### PR DESCRIPTION
**Pull request by sarahamilton**
Hi! I got a bit confused in the instructions for this sample, so made some slight edits to clarify the steps for testing complications.

### CODE REVIEW

File: WearComplicationProvidersTestSuite/README.md [hunk 1]
```diff
@@ -16,9 +16,8 @@ Therefore, you may see an error next to the 'Run' button. To fix, click on the
 "Wearable" dropdown next to the 'Run' button and select 'Edit Configurations'. Under the
 'Launch Options', change the 'Launch' field from 'Default APK' to 'Nothing' and save.
 
-This sample provides dummy data for testing the complications UI in your watch face. After
-selecting a type from your watch face configuration Activity, you can tap on the complications to
-see more options.
+This sample provides dummy data for testing the complications UI in your watch face. To 
+select the type of complication you want to use on your watch face, either double-tap the complication (emulator) or long-press the complication (on a Wear 3 device) to access complication settings. Choose the type of complication you would like from the WearComplicationProviderTestSuite provider's list. Once you choose the complication type(s) you would like, you can tap on the complications to see more options.
```
Comment (codingjeremy) [inline WearComplicationProvidersTestSuite/README.md:20]: Split over multiple lines so it meets column restrictions.

Also, Wear 3 = latest Developer Preview of Wear OS.

**Review by codingjeremy — COMMENTED** [review]
Just a couple small nits.

File: WearComplicationProvidersTestSuite/README.md (hunk 1 above)
Comment (castedmo) [reply WearComplicationProvidersTestSuite/README.md:20]: AFAIK configuring complications (without going through a configuration activity), is implementation dependant. So I think this update is a little bit misleading.

Comment (sarahamilton) [reply WearComplicationProvidersTestSuite/README.md:20]: Ah, sorry if I am misunderstanding -- what is the "configuration activity" in this case? And how would you access it?

Comment (castedmo) [reply WearComplicationProvidersTestSuite/README.md:20]: The one you get when you long press on a watch face and click on the cog icon below it.

**Review by ColleenKeegan — CHANGES_REQUESTED** [review]
****

**Comment by yschimke** [comment]
@sarahamilton going to close as it's not clear from the conversation that it's always correct.  Re-open and resolve if you want to land it still.

**Pull request closed without merging**